File: Cap 10/10.5 - Escopo de Variaveis/Troca Valores V1/tricaval1.rb
=begin
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 * 
 * Capitulo ..: 10 - Topico 10.5
 * 
 * Programa ..: TROCA: CONVENCIONAL
=end

puts 'TROCA DE VALORES - V1'
puts

print 'Entre valor para variavel <A>: '
a = gets.chomp.to_i

print 'Entre valor para variavel <B>: '
b = gets.chomp.to_i

x = a
a = b
b = x

puts
puts 'Os valores trocados sao:'
puts
puts "<A> = #{a}"
puts "<B> = #{b}"

File: Cap 10/10.5 - Escopo de Variaveis/Troca Valores V1/tricaval1.vb
' Livro .....: Algoritmos
'              Logica para Desenvolvimento de Programacao de 
'              Computadores
' Autores ...: Jose Augusto N. G. Manzano
'              Jayr Figueiredo de Oliveira
' Editora ...: Erica
' 
' Capitulo ..: 10 - Topico 10.5
'  
' Programa ..: TROCA: CONVENCIONAL

Module Module1

    Sub Main()
        Dim X As Integer
        Dim A As Integer
        Dim B As Integer

        Console.WriteLine("TROCA DE VALORES - V1")
        Console.WriteLine()

        Console.Write("Entre valor para variavel <A>: ")
        A = Console.ReadLine()

        Console.Write("Entre valor para variavel <B>: ")
        B = Console.ReadLine()

        X = A
        A = B
        B = X

        Console.WriteLine()
        Console.WriteLine("Os valores trocados sao:")
        Console.WriteLine()
        Console.WriteLine("<A> = " & A)
        Console.WriteLine("<B> = " & B)

    End Sub

End Module
